source/opt/inline_pass.cpp

```diff
@@ -266,6 +611,14 @@ bool InlinePass::GenInlineCode(
     return false;
   }
 
+  // Inline instructions of the caller function up to the call instruction.
+  uint32_t returnLabelId = 0;
+  std::unique_ptr<BasicBlock> new_blk_ptr = InlineInstsBeforeEntryBlock(
+      new_blocks, &callee2caller, &preCallSB, &single_trip_loop_cont_blk,
+      &returnLabelId, call_inst_itr, call_block_itr, caller_is_loop_header,
+      calleeFn);
```

Comment:
I like this, but it does have a lot of parameters.  We should see if we can reduce the number of parameters.

Reply:
I removed this function to reduce the number of arguments.
Instead, I inlined the contents of this function.